sdk/keyvault/keyvault-certificates/.eslintrc.json

```diff
@@ -4,6 +4,8 @@
   "ignorePatterns": ["src/core"],
   "rules": {
     "@typescript-eslint/no-this-alias": "off",
-    "no-invalid-this": "off"
+    "no-invalid-this": "off",
+    "@azure/azure-sdk/ts-package-json-module": "warn",
```

Comment:
does this mean we have active warnings now?

Reply:
I'm not sure what you mean. This change is needed because the current rules don't work with this project structure. Once this is merged, and after we polish it, I think we will have a good enough idea of how to change the ESLint rules! Let me know what you think.

Reply:
I guess I don't see the point in leaving active warnings in a project. Either we should fix the issue (which in this case I think means fixing the rule?) or disable the rule for now and file an issue to follow-up on turning it back on.

Reply:
(@willmtemple)

Here's the thing:

As we move forward with the test guidelines, we will need to update several things, including the eslint rules. It will take us some time to get there though. I think we would better fix the libraries first, then fix the eslint plugin. What Will did makes sense for our current state. This PR is in an intermediate state.